src/ontology/cl-edit.owl

```diff
@@ -33380,6 +33381,17 @@ AnnotationAssertion(rdfs:label obo:CL_4052029 "subpleural fibroblast")
 SubClassOf(obo:CL_4052029 obo:CL_0000057)
 SubClassOf(obo:CL_4052029 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048))
 
+# Class: obo:CL_4052030 (adventitial fibroblast)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28705796") Annotation(oboInOwl:hasDbXref "PMID:36718802") obo:IAO_0000115 obo:CL_4052030 "A fibroblast of the adventitia of a blood vessel. This cell contributes to vascular homeostasis, remodeling, and inflammation by producing extracellular matrix components, cytokines, and growth factors. Adventitial fibroblast can transition into an activated state during injury or disease, marked by increased proliferation, migration, matrix deposition, and contractile protein expression")
+AnnotationAssertion(terms:contributor obo:CL_4052030 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052030 "2024-12-05T11:01:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36718802") rdfs:comment obo:CL_4052030 "Single-cell transcriptomics identifies two adventitial fibroblast-specific markers, PDGFRA and DPEP1, highlighting fibroblast heterogeneity in health and cardiovascular disease (CVD) in humans and mice.")
```

Comment:
I think it would be good to specify the tissue (aorta). Also, they validated the presence of protein of these two genes by IH and FC.